src/test/groovy/graphql/execution/incremental/DeferExecutionSupportIntegrationTest.groovy

```diff
@@ -1604,6 +1604,37 @@ class DeferExecutionSupportIntegrationTest extends Specification {
         incrementalResults.any { it.incremental[0].path == ["posts", 2] }
     }
 
+
+    def "two fragments one with defer and one without"() {
+        given:
+        def query = '''
+            query {
+                post {
+                 text
+                ...f1 @defer
+                ...f2
+                }
+            }
+            
+         fragment f1 on Post {
+               summary
+          }
+         fragment f2 on Post {
+           summary
+          }
+        '''
+        when:
+        def initialResult = executeQuery(query)
+
+        then:
+        initialResult.toSpecification() == [
+                data: [post: [summary: "A summary", text: "The full text"]]
+        ]
+
```

Comment:
can you assert on the incremental payloads as well?

Reply:
What do you want to assert exactly? there is no incremental result in this case: is that correct?

Should we add another test case with some incremental result?

Reply:
You're right. I misread your test scenario.
In that case, do we need to assert that there aren't incremental payloads? Like, `initialResult` is not `IncrementalExecutionResult`? 
And perhaps change the name of the variable, `initialResult` gives the impression that this is an incremental result.